ebsco/edsapi-koha-plugin
# Corrects Safari Issues

When a user loads Koha on Safari on iOS, it was not selecting the right item in the rebuilt dropdown that the Koha EDS Plugin creates.  This fix changes a few factors on how that dropdown gets built to resolve this issue.

To Test:
On an iOS device, preferably a phone or tablet, open Safari.
***Note that you may need to first clear your cache if you've recently navigated to this page as Safari appears to have a very long term cache of javascript files.
Navigate to http://mike-koha.ebsco-gss.net/
Verify after load that the dropdown has "Catalogue" selected.
Verify that you can click the dropdown and select "Switch to Discovery".
This should change the option to say "Discovery" and prompt you that you are able to search EDS.
Do a search to verify it's returning EDS results.
On the results page, select "Switch to Catalogue" and execute another search.
Verify that on the resulting page, "Catalogue" is selected in the dropdown and "Switch to Discovery" is an option.
